python/pyspark/sql/connect/column.py

```diff
@@ -352,17 +365,332 @@ def __init__(self, expr: Expression) -> None:
     __rpow__ = _bin_op("pow", reverse=True)
     __ge__ = _bin_op(">=")
     __le__ = _bin_op("<=")
-    # __eq__ = _bin_op("==")  # ignore [assignment]
+
+    _eqNullSafe_doc = """
+        Equality test that is safe for null values.
+
+        .. versionadded:: 2.3.0
```

Comment:
```suggestion
        .. versionadded:: 3.4.0
```

Reply:
In fact, we should remove all `versionadded:: 3.4.0` in these pleaces, and just add one `versionadded:: 3.4.0` into the top level of the package. We can just remove them for now.

Reply:
I chose to remove all `versionadded` from this PR. 

Later we can go through all the places of Connect probably to remove `versionadded:: 3.4.0` but only have package level comment, given Connect is a new thing since 3.4.